pyEpiabm/pyEpiabm/place.py

```diff
@@ -0,0 +1,89 @@
+#
+# Place Class
+#
+from .person import Person
+import typing
+from .place_type import PlaceType
+
+
+class Place:
+    """Creates a place class which represents spaces such
+    as cafes, restaurants and hotel where people may come
+    into contact with others outside their household.
+    """
+    def __init__(self, loc: typing.Tuple[float, float],
+                 place_type: PlaceType, cell, microcell):
+        """Constructor method.
+
+        :param loc: (x,y) coordinates of the place.
+        :type loc: tuple
+        :param place_type: Categorises the place.
+        :type place_type: 'PlaceType' enum
+        :param cell: An instance of :class:`Cell`
+        :type cell: Cell
+        :param microcell: An instance of :class:`Microcell`
+        :type microcell: Microcell
+        """
+        self._location = loc
+        self.persons = []
+        self.place_type = place_type
+        self.max_capacity = 50
```

Comment:
Is this something CovidSim sets that we are mimicking? Or just a reasonable value if we always anticipate changing it?

Reply:
Reasonable value we anticipate changing